examples/01-Detailed-Examples/05-mesh-exploration.py

```diff
@@ -0,0 +1,210 @@
+# Get ``Simulation`` object
+# -------------------------
+# Get the ``Simulation`` object that allows access to the result. The ``Simulation``
+# object must be instantiated with the path for the result file. For example,
+# ``"C:/Users/user/my_result.rst"`` on Windows or ``"/home/user/my_result.rst"``
+# on Linux.
+
+example_path = examples.download_all_kinds_of_complexity()
+simulation = post.StaticMechanicalSimulation(example_path)
+
+# print the simulation to get an overview of what's available
+print(simulation)
+stress_df = simulation.stress(components=["XX"])
+print(stress_df)
+
+###############################################################################
+# Get the mesh
+# ------------
+
+# Retrieve the actual mesh
+mesh = simulation.mesh
+
+###############################################################################
+# Query basic information about the mesh (available in PyDPF-Core)
+# --------------------------------------
+
+# Node IDs
+n_ids = mesh.node_ids
+
+# Element IDs
+e_ids = mesh.element_ids
+
+# Available named selection names
+named_selections = mesh.available_named_selections
+
+# Number of nodes
+n_nodes = mesh.num_nodes
+
+# Number of elements
+n_elements = mesh.num_elements
+
+# Number of named selections
+n_ns = len(named_selections)
+
+# Unit (get and set)
+mesh_unit = mesh.unit
+mesh.unit = "mm"
+
+print(n_ids)
+print(e_ids)
+print(named_selections)
+print(n_nodes)
+print(n_elements)
+
+###############################################################################
+# Get Named Selections
+# --------------------
+ns_list     = mesh.available_named_selections
+first_key = ns_list[0]
+named_selection = mesh.named_selections[first_key]
+
+for k in mesh.named_selections.keys():
+    print(k)
+for v in mesh.named_selections.values():
+    print(v)
+for k, v in mesh.named_selections.items():
+    print(f"{k} = {v}")
+
+###############################################################################
+# Get elements
+# ------------
+
+# Get an element by ID
+el_by_id = mesh.elements.by_id[1]
+
+# Get an element by index
+index = el_by_id.index
+print(mesh.elements[index])
+
+###############################################################################
+# Element Types and Materials
+# ---------------------------
+
+# Get the element types
+el_types = mesh.element_types
+print(el_types)
+
+# Get the materials
+e_materials = mesh.materials
+print(e_materials)
+
+###############################################################################
+# Query information about one particular element
+# ----------------------------------------------
+
+# Get the nodes of an element
+mesh.elements[1].nodes
+
+# Get the node IDs of an element
+mesh.elements[1].node_ids
+
+# Get the nodes of an element
+mesh.elements[1].n_nodes
+
+# Get the type of the element
+mesh.elements[1].type_info
+mesh.elements[1].type_id
+
+# Get the shape of the element
+mesh.elements[1].shape
```

Comment:
@ansys-akarcher It may be a better idea to illustrate here that mesh.elements[] is by index, thus zero-based.
Illustrating with "1" means there can be a doubt IMO.
```suggestion
mesh.elements[0].nodes

# Get the node IDs of an element
mesh.elements[0].node_ids

# Get the nodes of an element
mesh.elements[0].n_nodes

# Get the type of the element
mesh.elements[0].type_info
mesh.elements[0].type_id

# Get the shape of the element
mesh.elements[0].shape
```